src/style.css

```diff
@@ -1,3 +1,67 @@
+html,
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+[data-qa='hover']:hover {
+  color: #00acdc;
+}
+
+:root {
+  --colorisacive: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 500;
+  padding: 0 50px;
+}
+
+.header__link {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  display: block;
+  position: relative;
+  height: 60px;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+nav ul li:not(:last-child) {
```

Comment:
to much selectors, just use item:not.....
make it in all parts ofr your code

Reply:
that is interestin... maybe I have copied bad links, as I have checked this before deploy and all tests were passed :)

ok I will fix selectors